scrapy/crawler.py

```diff
@@ -142,10 +140,8 @@ def crawl(self, *args: Any, **kwargs: Any) -> Generator[Deferred[Any], Any, None
         if self.crawling:
             raise RuntimeError("Crawling already taking place")
         if self._started:
-            warnings.warn(
-                "Running Crawler.crawl() more than once is deprecated.",
-                ScrapyDeprecationWarning,
-                stacklevel=2,
+            raise RuntimeError(
+                "Running Crawler.crawl() more than once on the same instance is forbidden."
```

Comment:
“Forbidden” in this context sounds a bit weird to me. What about:

```suggestion
                "Cannot run Crawler.crawl() more than once on the same instance."
```